Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -89,6 +89,14 @@ final class Analytics: AnalyticsProtocol {
             .property(OnboardingCardNTPExperiment.analyticsProperty))
     }
     
+    func ntpSeedCounterExperiment(_ action: Action, seedCounterManager: SeedProgressManagerProtocol.Type) {
+        track(Structured(category: Category.climateCounter.rawValue,
+                         action: action.rawValue)
+            .label(Label.Browser.newTab.rawValue)
+            .value(NSNumber(integerLiteral: seedCounterManager.loadTotalSeedsCollected()))
```

Comment:
I guess it makes more sense to directly receive the `NSNumber` here instead of the full protocol type? That way the Analytics class is not responsible for converting and leaves that to whoever calls it.

Reply:
Good spot 👍 I like it. So we can convert the `NSNumber` at upper level and injecting here something like `seeds: NSNumber = nil)`